pymatgen/io/lobster/inputs.py

```diff
@@ -116,176 +115,139 @@ class Lobsterin(UserDict, MSONable):
         "kpointwiseSpilling",
         "LSODOS",
     )
-    # several of these keywords + ending can be used in a lobsterin file:
-    LISTKEYWORDS = ("basisfunctions", "cohpbetween", "createFatband")
+    # These keywords + ending can be used in a lobsterin file
+    _LIST_KEYWORDS: tuple[str, ...] = (
+        "basisfunctions",
+        "cohpbetween",
+        "createFatband",
+    )
 
-    # all keywords known to this class so far
-    AVAILABLE_KEYWORDS = FLOAT_KEYWORDS + STRING_KEYWORDS + BOOLEAN_KEYWORDS + LISTKEYWORDS
+    # Generate {lowered: original} mappings
+    FLOAT_KEYWORDS: ClassVar = {key.lower(): key for key in _FLOAT_KEYWORDS}
+    STRING_KEYWORDS: ClassVar = {key.lower(): key for key in _STRING_KEYWORDS}
+    BOOLEAN_KEYWORDS: ClassVar = {key.lower(): key for key in _BOOLEAN_KEYWORDS}
+    LIST_KEYWORDS: ClassVar = {key.lower(): key for key in _LIST_KEYWORDS}
 
-    def __init__(self, settingsdict: dict):
+    # All known keywords
+    AVAILABLE_KEYWORDS: ClassVar = {**FLOAT_KEYWORDS, **STRING_KEYWORDS, **BOOLEAN_KEYWORDS, **LIST_KEYWORDS}
```

Comment:
is this up-to-date with lobster5?

Reply:
I think no... Upon checking the manual, LOBSTER 5.0.0 might have introduced the following additional keywords:
> printLmosOnAtoms Followed by indices of targeted atoms, this keyword will
calculate embedded localized molecular orbitals as formu-
lated in M. Pauls, D. Schnieders, R. Dronskowski, J. Phys.
Chem. A 2023, 127, 6541–6551 and print those centered on
the requested atoms to file in a cube file format.

> printTotalSpilling This keyword will instruct LOBSTER to print the
absolute total spilling next to the absolute charge
spilling.

> customSTOforAtom This keyword allows to define custom STO basis
sets for atoms. The syntax is as follows:
useBasisSet custom
customSTOforAtom [ELEMENT] [BASIS_FILE]
Where [ELEMENT] needs to be replaced by the ele-
ment symbol and [BASIS_FILE] is a basis set file
residing within the working directory that defines
the basis. An example of such basis set file is
shipped with every copy of LOBSTER.

Reply:
this might be something worth to check carefully as e.g `cobibetween` is available since lobster 4.0.0

Reply:
Several might be missing. I did only update those that I did not need for the automation.

Reply:
Added in 08d48164bf2e21588d41c20bec8a75a25b1d38e1 and a11423243f4369d108700f3d0a905a154460b819. Let me know if anything is missing or I'm adding to the wrong section :)